charts/README.md

```diff
@@ -0,0 +1,31 @@
+Deploy CSI Baremetal to kind-cluster using operator
+---------------------
+
+Last update: 26.03.2021
+
+! Kind-cluster has pre-loaded csi images
+
+1. Set CSI version
+
+```
+export csiVersion=0.0.13-375.3c20841
```

Comment:
Why do we need to mention specific version in README.md?

Reply:
remove